wordpress.org/public_html/wp-content/plugins/wporg-gp-plugin-directory/inc/sync/class-translation-sync.php

```diff
@@ -135,8 +139,9 @@ public function sync_plugin_translations_on_commit( $args ) {
 	public function queue_translation_for_sync( $translation ) {
 		global $wpdb;
 
-		// Only propagate current translations without warnings.
-		if ( 'current' !== $translation->status || ! empty( $translation->warnings ) ) {
+		$allowed_statuses = array( 'current', 'rejected', 'changesrequested', 'fuzzy' );
```

Comment:
If we sync the `changesrequested` status, I think the rejection reason will not be visible. We could implement something like on bulk reject where there is a "linking comment". We shouldn't do this here but we could add a `do_action( 'translation_synced', $translation );` in anticipation of that.

Reply:
Probably some additional parameters would be useful like source and target project.